project/src/main/java/com/google/sps/data/NewsArticle.java

```diff
@@ -25,15 +25,21 @@
   private String publisher;
   private Date publishedDate;
   private String content;
+  private String abbreviatedContent;
+  private String summarizedContent;
+  private int priority;
   private NewsArticleCategory category;
 
   public NewsArticle(String title, String url, String publisher, Date publishedDate,
-      String content) {
+      String content, String abbreviatedContent, String summarizedContent, int priority) {
```

Comment:
It would be helpful to include a comment about how abbreviatedContent is different from summarizedContent